### PR TITLE
feat: fetch mdn-browser-compat-data from a remote source

### PR DESCRIPTION
**Pull request by ahoneiser**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,10 +2,8 @@ FROM node:14-slim
 
 WORKDIR /app
 
-# Install dependencies, but don't generate lock files as we want to always fetch
-# the lastest @mdn/browser-compat-data package.
 COPY package.json ./
-RUN npm install --no-package-lock && \
+RUN npm install --no-dev && \
```
Comment (juliandescottes) [inline Dockerfile]: I can't find `--no-dev` in the documentation of npm install, maybe we can just do `npm install`

File: package.json [hunk 2]
```diff
@@ -5,8 +5,9 @@
     "ingest": "node update_remote_settings_records.mjs"
   },
   "devDependencies": {
-    "@mdn/browser-compat-data": "latest",
-    "btoa": "1.2.1",
-    "node-fetch": "^3.2.0"
+    "btoa": "1.2.1"
+  },
+  "dependencies": {
+    "node-fetch": "^3.2.10"
```
Comment (juliandescottes) [inline package.json:9]: nit: I think we can keep everything in `devDependencies`. If I remember correctly, dependencies are only relevant if you are publishing a package (ie dependencies are bundled in the package, devDependencies are not). It's not our case so we should be fine only with devDependencies.

File: update_remote_settings_records.mjs [hunk 3]
```diff
@@ -21,7 +21,9 @@ import btoa from "btoa";
 // If we only support Node 16+, can be updated to:
 //   import bcd from '@mdn/browser-compat-data' assert { type: 'json' };
 // See https://github.com/mdn/browser-compat-data.
-import compatData from "@mdn/browser-compat-data/forLegacyNode";
+//import compatData from "@mdn/browser-compat-data/forLegacyNode";
```
Comment (juliandescottes) [inline update_remote_settings_records.mjs]: remove the commented out line

**Review by juliandescottes — CHANGES_REQUESTED** [review]
Thanks a lot for implementing a fix! This looks straightforward, just a few comments to address before merging.

File: Dockerfile (hunk 1 above)
Comment (ahoneiser) [reply Dockerfile]: thanks, reverted changes :+1:

File: package.json (hunk 2 above)
Comment (ahoneiser) [reply package.json:9]: sounds about right, updated `package.json`

File: update_remote_settings_records.mjs (hunk 3 above)
Comment (ahoneiser) [reply update_remote_settings_records.mjs]: thanks, it's gone :+1:

File: Dockerfile [hunk 4]
```diff
@@ -2,15 +2,13 @@ FROM node:14-slim
 
 WORKDIR /app
 
-# Install dependencies, but don't generate lock files as we want to always fetch
-# the lastest @mdn/browser-compat-data package.
 COPY package.json ./
-RUN npm install --no-package-lock && \
-  npm cache clear --force
+RUN npm install --no-package-lock; \
+  npm cache clean --force
```
Comment (juliandescottes) [inline Dockerfile:7]: oh good catch for the `clear` typo!

File: Dockerfile [hunk 5]
```diff
@@ -2,15 +2,13 @@ FROM node:14-slim
...
 
 # copy sources
 COPY update_remote_settings_records.mjs ./
 COPY ./version.json ./
 
 # set CMD
-CMD ["npm", "run", "ingest"]
+CMD ["node", "update_remote_settings_records.mjs"]
```
Comment (juliandescottes) [inline Dockerfile:14]: We are no longer using `ingest`, so I'm not sure if we should keep it in the package.json?

**Review by juliandescottes — APPROVED** [review]
LGTM, thanks for the fix! One nit about the now unused "ingest" script

File: Dockerfile (hunk 5 above)
Comment (ahoneiser) [reply Dockerfile:14]: oh, good catch, forgot to clean that file up .. will fix this with another PR

**Pull request merged**